js/layout-javascript/news-feed-code.js

```diff
@@ -1636,7 +1636,9 @@ DynamicList.prototype.searchData = function(options) {
       }
 
       if (limitEntriesEnabled) {
-        _this.$container.find('.limit-entries-text')[truncated && _this.data.limitEntries > 0 ? 'removeClass' : 'addClass']('hidden');
```

Comment:
The `truncated` variable from the results of `_this.Utils.Records.runSearch()` should be able to determining this. I believe `truncated` should only be `true` if search, filter and bookmarks are all disabled. There should therefore be no need to separately calculate a `showLimiText` status.

Reply:
fixed